src/lib/numbers/__tests__/clamp.test.ts

```
@@ -0,0 +1,52 @@
+    expect(clamp(2.5, { min: 1.2, max: 3.4 })).toBeCloseTo(2.5);
+    expect(clamp(0.5, { min: 1.2, max: 3.4 })).toBeCloseTo(1.2);
+    expect(clamp(5.6, { min: 1.2, max: 3.4 })).toBeCloseTo(3.4);
+  });
+
+  it("handles Infinity and -Infinity correctly", () => {
+    expect(clamp(Infinity, { min: -100, max: 100 })).toBe(100);
+    expect(clamp(-Infinity, { min: -100, max: 100 })).toBe(-100);
+  });
+
+  it("propagates NaN", () => {
+    expect(Number.isNaN(clamp(NaN, { min: 0, max: 10 }))).toBe(true);
```

Biome (1.9.4)

[error] Use Number.NaN instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NaN instead.


(lint/style/useNumberNamespace)